<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventRentDialog</class>
 <widget class="QDialog" name="EventRentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>268</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>249</width>
    <height>287</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Film:</string>
   </property>
  </widget>
  <widget class="QLabel" name="eventLabel">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>20</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DÃ¡tum:</string>
   </property>
  </widget>
  <widget class="QLabel" name="datumLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rentButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Foglal</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Terem:</string>
    </property>
   </widget>
   <widget class="QLabel" name="teremLabel">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>70</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Oszlop:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sorComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="oszlopComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>label</zorder>
  <zorder>eventLabel</zorder>
  <zorder>label_4</zorder>
  <zorder>datumLabel</zorder>
  <zorder>rentButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
